resources/lang/en/accounts.php

```diff
@@ -147,6 +147,20 @@
         'update' => 'update',
     ],
 
+    'user_totp' => [
+        'title' => 'Authenticator App',
+
+        'button' => [
+            'remove' => 'Remove',
+            'setup' => 'Setup',
```

Comment:
```suggestion
            'setup' => 'Set up',
```

(assumes this is referring to "Set up an authenticator app")

Reply:
For this specific instance I'd probably change to "Add authenticator app" or something, rather than "Set up".